### DISCUSSION TITLE
Surface grid

### DISCUSSION THREAD

**Discussion by azkaron87** [acoular/acoular#406] [Q&A] [answered]
Is there a way to generate a grid in the shape of surface, such as half-sphere?

**Comment by esarradj** [comment, marked as answer]
One option would be to create the xyz co-ordinates of the grid points with whatever external tool or Python function you like and then to use `ImportGrid`. There, you could either import the grid from a file (see the docs) or you could not import any file, but set the `gpos_file` attribute directly with an array of all  xyz co-ordinates. 
Note that from version 25.01 onwards this will be the `pos` attribute instead.
